Makefile

```diff
@@ -160,7 +160,7 @@ min-test:
 TO_CLEAN=$(foreach ext,checked checked.lax out cmd err time dump types.vaf krml cmx cmo cmi o d a,-or -iname '*.$(ext)')
 clean:
 	find . -iname '.depend*' $(TO_CLEAN) -delete
-	rm -rf obj/*
+	find obj -depth 1 -name .gitignore -o -delete
```

Comment:
is this correct? I feel like there's a `-not` missing

Reply:
Updated to avoid non-portable "-depth n", and change logic to use -not and implicit and instead of -o.

Reply:
Output from configure:
```
... detected x64 platform
...  supports compilation of 128-bit and 256-bit AVX/AVX2
...  supports vale
... not using known buggy Apple LLVM build
...  supports our inline ASM
... using a non-buggy GCC
 supports _addcarry_u64
... found ocamlfind in /usr/local/bin/ocamlfind
ocamlfind: Package `ctypes' not found
OCaml package ctypes not found
OCaml bindings disabled
```

Reply:
Does this PR need any more input from me?